designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/UsageStatisticsReportsSettingsService.scala

```diff
@@ -177,7 +177,7 @@ private[statistics] case class ScenarioStatisticsInputData(
     processingMode: ProcessingMode,
     deploymentManagerType: DeploymentManagerType,
     // For fragments status is empty
-    status: Option[StateStatus],
+    status: Option[String],
```

Comment:
It is changed to `String` because statistics mechanism is based on DTOs, where after changes is only scenario status name.